app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -233,7 +233,7 @@ def hint_tag(attribute, options)
     def error_tag(attribute, options)
       return unless error?(attribute, options)
 
-      message = object.errors[attribute].join(', ')
+      message = object.errors[attribute].first
```

Comment:
Any reason why we just take the first error?

Reply:
If you look at most of the designs, there is a consistent effort to avoid multiple errors per attribute. Mostly that's done via the form, but often it's done with custom validation (it why some forms have more complicated validation than would normally be used). 

This is a simpler solution.

Reply:
Ok. Fair enough. I was thinking of the story with the file upload I'm working on at the moment.
For example you could potentially have these 2 errors for the same attribute:
- The selected file must be a DOC, DOCX, RTF, [anything else we accept..], JPG, BMP, PNG, TIF or PDF
- The selected file must be smaller than [xxx]MB

I thought it would be usefull to show both